### PR TITLE
moving to hub v1.1

### PR DESCRIPTION
**Pull request by mhshami01**
https://msazure.visualstudio.com/One/_workitems/edit/9530509

Update default edgeHub image to version 1.1.

### CODE REVIEW

**Comment by cindydeng1998** [comment]
Have you updated the version number of this release? If so, I would need to update it in iotedgedev

**Comment by mhshami01** [comment]
> Have you updated the version number of this release? If so, I would need to update it in iotedgedev

Not yet - version change will be part of the 'release' step later.

**Review by marianan — APPROVED** [review]
:shipit:

**Review by cindydeng1998 — APPROVED** [review]

**Review by smcd253 — COMMENTED** [review]
is there support for py36 and py37? iotedgedev implements this tool with both 36 and 37

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist=py27, py37
+envlist=py36
```
[the remark this answers is not in the document]
Comment (smcd253) [reply tox.ini]: out of scope for this PR, but we should address this

**Review by smcd253 — APPROVED** [review]
lgtm

**Pull request merged**